### DISCUSSION TITLE
Integration with vuejs 3 / Nuxt 3

### DISCUSSION THREAD

**Discussion by underthesand** [ajaxorg/ace#5295] [Q&A]
the tutorials available on the net are outdated, after having tried several options (vue package , ace-builds or integration with external script) I notice that the integration is a headache and that the editor is buggy when it is associated with vuejs or nuxt (clipping, slow, unwanted reload).

If anyone has managed to successfully integrate it into production I would like to have their feedback. Thanks in advance

**Comment by mkslanc** [comment]
Hi @underthesand ! Could you provide more specific details, which parts are not working? I used "nuxt":"3.7.4", "vue": "3.3.4", "ace-builds":"1.16" and a little bit modified (to load all dynamic modules and ace-linters) [vue-3-ace-editor](https://github.com/CarterLi/vue3-ace-editor). From my side it seems working quite good.

**Reply by mkslanc** [reply]
[screen-capture (41).webm](https://github.com/ajaxorg/ace/assets/32402726/b6b90675-a75d-4ea1-9345-4c9bb9fedaf0)
